posthog/api/decide.py

```diff
@@ -159,5 +159,19 @@ def get_decide(request: HttpRequest):
                 on_permitted_recording_domain(team, request) or not team.recording_domains
             ):
                 response["sessionRecording"] = {"endpoint": "/s/"}
+
+            source_files = PluginSourceFile.objects.filter(
+                plugin__pluginconfig__team=team,
+                plugin__pluginconfig__enabled=True,
+                filename="web.ts",
+                status=PluginSourceFile.Status.TRANSPILED,
+            ).all()
+            if len(source_files) > 0:
+                response["inject"] = []
+                for source_file in source_files:
+                    response["inject"].append(
+                        {"id": source_file.plugin_id, "source": source_file.transpiled, "payload": {}}
```

Comment:
How big would these files be? 

Worried if we manage to bloat decide response to 100kb this will mean a significant amount of traffic dropped for e.g. mobile.

Reply:
We definitely need *some* limits here. My idea was that if the code goes over some limit (1kb? not sure), then instead of sending over that long block of code, we'd send a short bootloader instead.